chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/DeveloperStatusTest.java

```diff
@@ -76,6 +77,9 @@
     @Autowired
     private DeveloperValidationFactory developerValidationFactory;
 
+    @Autowired
+    private FF4j ff4j;
```

Comment:
I don't see this used anywhere in the file

Reply:
Good catch - was part of an older impl. Will remove. Although, won't be surpised if it needs to be added back for new tests.